server/models/schema.sql

```diff
@@ -1722,10 +1718,13 @@ CREATE TABLE `taxe_ipr` (
 DROP TABLE IF EXISTS `transaction_type`;
 
 CREATE TABLE `transaction_type` (
-  `id` tinyint(3) unsigned NOT NULL AUTO_INCREMENT,
-  `service_txt` varchar(45) NOT NULL,
+  `id` tinyint(3) unsigned NOT NULL,
```

Comment:
It's probably better to make this `AUTO INCREMENT`.  That way multiple users do not collide when creating a transaction type.